chapters/external-repository-identifiers.md

```diff
@@ -238,15 +238,41 @@ Examples:
 
 External Reference Site: [Software Heritage persistent IDentifiers](https://docs.softwareheritage.org/devel/swh-model/persistent-identifiers.html)
 
-### F.4.1 gitbom <a name="gitbom"></a>
+### F.4.2 gitoid <a name="gitoid"></a>
 
-Locator Format: gitoid":"\<git object type>":"\<hash algorithm>":"\<hash value>
-	
-Contextual Example: gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64
+Locator Format: `"gitoid:"\<git object type>":"\<hash algorithm>":"\<hash value>`
+
```

Comment:
You may want to look at this in preview, as it appears that the `<` does not require escaping in a pre block.